### PR TITLE
GH-256 Allows merchants to delete paywall from dashboard.

### PR DESCRIPTION
**Pull request by milindmore22**
related to #256

### CODE REVIEW

**Comment by milindmore22** [comment]
@tomasstark Updated code to use `RevGenModal` please review

File: assets/src/js/pages/revenue-generator-dashboard.js [hunk 1]
```diff
@@ -24,7 +24,16 @@ import { RevGenModal } from '../utils/rev-gen-modal';
 				paywallPreview: '.rev-gen-dashboard-content-paywall-preview',
 				paywallContentWrapper: '.rev-gen-dashboard-content-paywall',
 
+				// Delete Paywall link.
+				removePaywallDashboard: '.rev-gen-dashboard-remove-paywall',
+				// Paywall remove warning modal.
+				paywallRemovalModal: '.rev-gen-preview-main-remove-paywall',
+				paywallRemove: '#rg_js_removePaywall',
+				paywallCancelRemove: '#rg_js_cancelPaywallRemoval',
```
Comment (tomasstark) [inline assets/src/js/pages/revenue-generator-dashboard.js:32]: Can these 3 now be removed with the latest refactor to use RevGenModal?

- paywallRemovalModal
- paywallRemove
- paywallCancelRemove

Comment (milindmore22) [reply assets/src/js/pages/revenue-generator-dashboard.js:32]: Yes, Good catch I missed those

**Comment by milindmore22** [comment]
Addressed PR feedback please review again

**Review by tomasstark — APPROVED** [review]
LGTM 🚀

**Pull request merged**